apps/fluent-tester/src/TestComponents/Avatar/BasicAvatar.android.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { FunctionComponent } from 'react';
+import { Avatar } from '@fluentui-react-native/avatar';
+import { View } from 'react-native';
+import { steveBallmerPhotoUrl } from './../PersonaCoin/styles';
+
+export const StandardUsage: FunctionComponent = () => {
+  return (
+    <View style={{ paddingBottom: 40 }} >
+      <View style={{
```

Comment:
For all the view style properties in this file, we should try to avoid creating the style object inline since that causes unnecessary rerenders. If we move the style object creation outside of the function component then there won't be unnecessary rerenders. 

Here's an [explanation](https://github.com/microsoft/fluentui-react-native/pull/2231#discussion_r995097257) Ruriko posted in one of my PRs:

> The way React components figure out if it needs to rerender is by shallow comparing all of its props and if anything is different, it will rerender the component. For objects (objects, arrays, functions), that means that if a new instance of that object is created and passed into the compare, then the shallow compare will fail because it's a different instance, even if the contents of the object are the same and so in reality no work needed to be done. You can think of it as a pointer comparison if you're used to native.
> 
> Anything inside of a (function) component is rerun when something asks components to rerender. So that means any instantiation of a const object will be redone, and new instances of those objects will be created. In this case exampleStrokeStyle has a new instance on every render and so the inner most view will be invalidated on every render.
>
> Moving the const object outside the components works because that code is only run once.

The other benefit is that there are a couple places where you have the same style declared several times, so that will fix the duplicated code

Reply:
Yeah, we have styles.ts in Common folder where shared styles can be added